tests/edwards_test_vector.rs

```diff
@@ -15,6 +15,8 @@ struct Derivation {
     expected_public_key: [u8; 32],
 }
 
+/// These test vectors were obtained by running HD derivation in another library
+/// that implements edwards HD derivation
```

Comment:
Which one though? :D

Reply:
You yourself suggested to never mention that library in our code!

Reply:
Ahh right. Forgot about that